quodlibet/po/fr.po

```diff
@@ -2728,49 +2735,47 @@ msgstr ""
 
 #: ../quodlibet/ext/playlist/export_to_squeezebox.py:73
 msgid "Export playlist to Squeezebox"
-msgstr ""
+msgstr "Exporter la liste de lecture vers Squeezebox"
 
 #: ../quodlibet/ext/playlist/export_to_squeezebox.py:74
 msgid "Playlist name (will overwrite existing)"
-msgstr ""
+msgstr "Le nom de la liste de lecture (écrase le nom précédent)"
 
 #: ../quodlibet/ext/playlist/export_to_squeezebox.py:91
 msgid "Export to Squeezebox playlist"
-msgstr ""
+msgstr "EXporter vers liste de lecture Sqeezebox"
 
 #: ../quodlibet/ext/playlist/remove_duplicates.py:20
-#, fuzzy
 msgid "Remove Playlist Duplicates"
-msgstr "Supprimer les doublons"
+msgstr "Supprimer les doublons dans la liste de lecture"
 
 #: ../quodlibet/ext/playlist/remove_duplicates.py:21
 msgid "Removes duplicate entries in a playlist."
-msgstr ""
+msgstr "Supprime les doublons dans une liste de lecture."
 
 #: ../quodlibet/ext/playlist/remove_duplicates.py:49
-#, fuzzy, python-format
+#, python-format
 msgid "Are you sure you want to remove %d duplicate song?"
 msgid_plural "Are you sure you want to remove %d duplicate songs?"
-msgstr[0] "Vous allez supprimer %d chanson."
-msgstr[1] "Vous allez supprimer %d chanson."
+msgstr[0] "Confirmer la suppression du titre %d en doublon ?"
+msgstr[1] "Confirmer la suppression des %d titres en doublon ?"
 
 #: ../quodlibet/ext/playlist/remove_duplicates.py:52
 #, python-format
 msgid "The duplicate songs will be removed from the playlist '%s'."
 msgstr ""
 
 #: ../quodlibet/ext/playlist/shuffle.py:15
-#, fuzzy
 msgid "Shuffle Playlist"
-msgstr "Créer une liste de lecture"
+msgstr "Lecture aléatoire"
```

Comment:
As previously mentioned, here it's about the Shuffle Playlist plugin which does not just play randomly but instead moves tracks randomly inside the playlist. So the playlist will be modified after that.